spec/tapioca/addon_spec.rb

```diff
@@ -0,0 +1,59 @@
+# typed: true
+# frozen_string_literal: true
+
+require "addon_spec_helper"
+require "ruby_lsp/ruby_lsp_rails/runner_client"
+require "minitest/hooks"
+
+module RubyLsp
+  module Tapioca
+    class AddonSpec < Minitest::HooksSpec
+      # The approach here is based on tests within the Ruby LSP Rails gem
+
+      # TODO: Replace by `before(:all)` once Sorbet understands it
```

Comment:
I don't think this TODO is very helpful but

```suggestion
      # TODO: Replace with `before(:all)` once Sorbet understands it
```